packages/fxa-auth-server/lib/senders/emails/templates/subscriptionAccountFinishSetup/index.stories.ts

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Meta } from '@storybook/html';
+import { subplatStoryWithProps } from '../../storybook-email';
+
+export default {
+  title: 'SubPlat Emails/Templates/subscriptionAccountFinishSetup',
+} as Meta;
+
+const createStory = subplatStoryWithProps(
+  'subscriptionAccountFinishSetup',
+  'Sent to a user to inform them that their payment is processing and prompt them to create a password and download subscription.',
```

Comment:
I think we can be _slightly_ more explicit here and specify it's for users that purchased the product without an existing Firefox account. They're taken through the passwordless flow and then sent this email.